CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 2.8.3)
-project(polhemus_ros)
+project(polhemus)
```

Comment:
Perhaps consider ``polhemus_driver`` or ``polhemus_ros_driver`` in line with over ROS drivers. This would setup a library ``libpolhemus`` which may be reserved for the original software which is not ROS-bound.